SpotMenu/AppDelegate.swift

```diff
@@ -95,10 +108,15 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         updateHidden()
     }
     
-    
-    
-    
+
     // MARK: - Popover
+    func openSite(_ sender: NSMenuItem) {
+//        NSApplicationDelegate.open(url, options: [:], completionHandler: nil)
```

Comment:
There is need NSApplicationDelegate to be import. I don't know how to import it property ;(